rust-src/concordium_base/src/updates.rs

```diff
@@ -836,8 +842,14 @@ impl Deserial for UpdatePayload {
             9u8 => Ok(UpdatePayload::BakerStakeThreshold(source.get()?)),
             10u8 => Ok(UpdatePayload::Root(source.get()?)),
             11u8 => Ok(UpdatePayload::Level1(source.get()?)),
-            12u8 => Ok(UpdatePayload::AddAnonymityRevoker(source.get()?)),
-            13u8 => Ok(UpdatePayload::AddIdentityProvider(source.get()?)),
+            12u8 => {
+                let _len: u32 = source.get()?;
+                Ok(UpdatePayload::AddAnonymityRevoker(source.get()?))
+            }
+            13u8 => {
+                let _len: u32 = source.get()?;
```

Comment:
Please document why the length is skipped.

Reply:
I have revised this.